tests/unit/drivers/test_eval_collect_driver.py

```diff
@@ -14,7 +15,7 @@ def input_function():
     ev2 = doc2.evaluations.add()
     ev2.value = 2
     ev2.op_name = 'op2'
-    return [doc1, doc2]
+    return [Document(doc1), Document(doc2)]
```

Comment:
remove the need for DocumentProto

Reply:
at the instantiation time

Reply:
yes `doc1 = DocumentProto()` can be replaced by `doc1 = Document()`